packages/core/README.md

```diff
@@ -0,0 +1,22 @@
+# TV Kitchen Appliance Core
+
+This repository contains core classes and constants for use by TV Kitchen Appliances and code that interacts with TV Kitchen Appliances.
+
+## Key Repository Contents
+
+This repository has some supporting classes, but the content that will be relevant to others are:
+
+* `AbstractAppliance` is a class which all TV Kitchen Appliances should extend and implement.
+
+## Implementing an Abstract Appliance
+
+An implemented TV Kitchen Appliance must override the following methods which are outlined in the [IAppliance interface](https://github.com/tvkitchen/base/blob/master/packages/interfaces/src/IAppliance.js):
+
+1. `getInputTypes`: returns an array of strings that represent the data types that the appliance accepts.
+2. `getOutputTypes`: returns an array of strings that represent the data types that the appliance produces.
+3. `isValidPayload`: returns a boolean indicating if a given `Payload` (instance of data) meets the conditions of the appliance.
+4. `invoke`: will actually process the data that has been collected by the appliance so far.
```

Comment:
There are a few more methods an appliance will need to define, so I think this list is out of date, but I think that's out of scope of this PR and will open a new issue. I'm just noting it to blunt the surprise when that issue comes in.

Reply:
I believe those are the only ones! (for now -- see [this impending doom discussion](https://github.com/tvkitchen/appliances/issues/19))

The rest of the IAppliance methods are implemented in AbstractAppliance.

Reply:
Oh, right right right. Sorry, I followed the link to the `IAppliance` and was like "wait there are more methods that throw `NotImplementedError`! But you're right, those three are (for now) defined by `AbstractAppliance` directly.

Sorry sorry resolved!